bridge/elrond/client.go

```diff
@@ -359,17 +390,22 @@ func (c *client) WasExecuted(_ context.Context, actionId bridge.ActionId, _ brid
 }
 
 // Sign will trigger the execution of a sign operation
-func (c *client) Sign(_ context.Context, actionId bridge.ActionId, _ *bridge.Batch) (string, error) {
+func (c *client) Sign(_ context.Context, actionId bridge.ActionId, batch *bridge.Batch) (string, error) {
 	builder := newBuilder(c.log).
 		Func("sign").
 		ActionId(actionId)
 
 	hash, err := c.sendTransaction(builder, c.gasMapConfig.Sign)
 
+	batchData, err := json.Marshal(batch)
+	if err != nil {
+		c.log.Warn("Elrond: error not critical while serializing transaction", "error", err)
+	}
+
 	if err == nil {
-		c.log.Info("Elrond: Signed", "hash", hash)
+		c.log.Info("Elrond: Signed", "hash", hash, "batch data", string(batchData))
 	} else {
-		c.log.Error("Elrond: Sign failed", "error", err.Error())
+		c.log.Error("Elrond: Sign failed", "batch data", string(batchData), "error", err.Error())
```

Comment:
optional: `err` instead of `err.Error()`

Reply:
done